.github/workflows/cmake_builds.yml

```
@@ -64,6 +64,23 @@ jobs:
         curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list | sudo tee /etc/apt/sources.list.d/msprod.list
         sudo apt-get update
         sudo ACCEPT_EULA=Y apt-get install -y msodbcsql17 unixodbc-dev
+        # HANA: client side
+        # Install hdbsql tool
+        curl -v -j -k -s -L -H "Cookie: eula_3_1_agreed=tools.hana.ondemand.com/developer-license-3_1.txt" https://tools.hana.ondemand.com/additional/hanaclient-latest-linux-x64.tar.gz --output hanaclient-latest-linux-x64.tar.gz \
+          && tar -xvf hanaclient-latest-linux-x64.tar.gz \
+          && sudo mkdir /usr/sap \
+          && sudo ./client/hdbinst -a client --sapmnt=/usr/sap \
+          && rm -rf client \
+          && rm hanaclient*
+        export PATH=/usr/sap/hdbclient:$PATH
+        # Download and compile odbc-cpp-wrapper
+        git clone https://github.com/SAP/odbc-cpp-wrapper.git \
+          && mkdir odbc-cpp-wrapper/build \
+          && cd odbc-cpp-wrapper/build \
+          && cmake .. \
```

Looking at the build log:
```
2022-03-31T12:41:15.1217328Z Cloning into 'odbc-cpp-wrapper'...
2022-03-31T12:41:15.5184370Z CMake Error at CMakeLists.txt:1 (CMAKE_MINIMUM_REQUIRED):
2022-03-31T12:41:15.5185153Z   CMake 3.12 or higher is required.  You are running version 3.9.1
```

```suggestion
          && /usr/bin/cmake .. \
```